src/app/views/sessions/signin/signin.component.ts

```diff
@@ -215,16 +215,7 @@ export class SigninComponent implements OnInit, OnDestroy, AfterViewInit {
   }
 
   checkBuildtime() {
-    const salt = (new Date()).getTime();
-    this.http.get(`./assets/buildtime?${salt}`, {responseType: 'text'}).subscribe((res) => {
-      const buildtime = res;
-      const previous_buildtime = window.localStorage.getItem('buildtime');
-      if (buildtime !== previous_buildtime) {
-        window.localStorage.setItem('buildtime', buildtime);
-        this._copyrightYear = this.localeService.getCopyrightYearFromBuildTime();
-        window.location.reload(true);
-      }
-    });
+    
```

Comment:
This results in a missing copyright year

![image](https://user-images.githubusercontent.com/29865743/110151199-301e3680-7dae-11eb-9cb1-4e25dccb750f.png)

Reply:
Need updates on middleware.
https://github.com/truenas/middleware/pull/6553

Reply:
It seems those changes have been merged on middleware side

Reply:
yeah, I am waiting for the building at this moment.

Reply:
@AlexKarpov98 Internal 149 is ready

Reply:
Updated